package.json

```diff
@@ -36,6 +39,7 @@
     "socket.io-client": "~2.2.0",
     "utf8": "3.0.0",
     "uuid": "^3.0.1",
+    "vaultclient": "scality/vaultclient#754b6e1",
```

Comment:
We introduce here a circular dependency: only cloudserver shall be able to have vaultclient in its package.json. The vaultclient lib instance shall be passed from cloudserver to Arsenal (just like sproxydclient and bucketclient)